### PR TITLE
Second Commit - Init angular

### PR DESCRIPTION
**Pull request by buiduong094**

### CODE REVIEW

File: client/app/ts/app.html5.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import {AboutComponent} from 'components/AboutComponent';
+
+/*
+ * Webpack
+ */
+require('css/styles.css');
+
+@Component({
+  selector: 'router-app',
+  template: `
```
Comment (octoberstorm) [inline client/app/ts/app.html5.ts]: better move to external html template

**Comment by PrinceNorin** [comment]
I think using [cli tool](https://github.com/angular/angular-cli) to generate project makes it cleaner. Is there any reason for not using it?

**Comment by octoberstorm** [comment]
@PrinceNorin I found it a little bit difficult to configure Webpack with the CLI, not looking into deeply yet. Did you face this?

**Comment by buiduong094** [comment]
@PrinceNorin have you generated Webpack by cli tool?

**Comment by PrinceNorin** [comment]
@octoberstorm most of configuration can be done by editing `.angular-cli.json` and if we really need a high level of customization we can use `eject` function, which will then spill out webpack config file and we can edit it to fit our need.

**Comment by PrinceNorin** [comment]
@buiduong094 yes I have and currently been using it in my own project as well

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,4 @@ angular2-kanban
 angular2-kanban.exe
 
 server/vendor/
+client/node_modules/
```
Comment (tuannd255) [inline .gitignore]: end file =)))))))

File: client/app/config/webpack.prod.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+var webpack = require('webpack');
+var webpackMerge = require('webpack-merge');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var commonConfig = require('./webpack.common.js');
+
+const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
```
Comment (tuannd255) [inline client/app/config/webpack.prod.js:6]: [Hỏi ngu] đống "=" đằng sau là khởi tạo hàng loạt hay nó thay cho || vậy?

Comment (ghost) [reply client/app/config/webpack.prod.js:6]: Cũng định hỏi như trên à (bow)

File: client/app/ts/app.component.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  <nav>
+    <a>Navigation:</a>
+    <ul>
+      <li><a [routerLink]="['home']">Home</a></li>
+      <li><a [routerLink]="['about']">About</a></li>
+    </ul>
+  </nav>
+
+  <router-outlet></router-outlet>
+</div>
```
Comment (ghost) [inline client/app/ts/app.component.html]: missing end file hay sao ý (bow)

**Comment by octoberstorm** [comment]
@PrinceNorin In that case, maybe we cannot use default `ng serve` or `ng test` anymore right?

File: client/app/config/webpack.prod.js (hunk 3 above)
Comment (buiduong094) [reply client/app/config/webpack.prod.js:6]: gán mà -_-

File: .gitignore (hunk 2 above)
Comment (buiduong094) [reply .gitignore]: (goodjob)

File: client/app/ts/app.component.html (hunk 4 above)
Comment (buiduong094) [reply client/app/ts/app.component.html]: (goodjob)

File: client/app/config/webpack.prod.js (hunk 3 above)
Comment (tuannd255) [reply client/app/config/webpack.prod.js:6]: dùng gán kiểu này khó hỉu à nghen !@!!

**Pull request merged**